src/rippled/payString.ts

```diff
@@ -0,0 +1,42 @@
+import axios from 'axios'
+/* eslint-disable import/no-unresolved -- Avoids 8kb gzipped of bloat and V.PayString code. */
+import { parsePayString } from '@paystring/utils/dist/parse'
+import { convertPayStringToUrl } from '@paystring/utils/dist/convert'
+/* eslint-enable import/no-unresolved */
+import logger from './lib/logger'
+
+const log = logger({ name: 'payString' })
+
+export interface PayStringResponse {
+  payString: string
+  addresses: {
+    paymentNetwork: string
+    environment: string
+    addressDetailsType: string
+    addressDetails: {
+      address: string
+      tag: string // optional
```

Comment:
Should this be written as `tag?: string` instead of this comment? or is the implication that it can be empty?

Reply:
Just confirmed the output from the server and it will be undefined.